marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/EntitySearchController.java

```diff
@@ -212,4 +212,15 @@ private EntitySearchService getEntitySearchService(String database) {
 
     public static class SavedQueries extends ArrayList<SavedQuery> {
     }
+
+    public static class LastProcessedMetadata {
+        public String lastProcessedByFlow;
+        public String lastProcessedByStep;
+        public String lastProcessedDateTime;
+        public ArrayList<DocumentSourceMetadata> sources;
```

Comment:
Let's call this "HubMetadata", as it's not all specific to when the document was last processed - for example, the sources are not specific to that concept.